website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -108,3 +108,25 @@ To delete a published tag to re-tag, follow these steps:
     git push -d origin vx.y.z
 ```
 
+## Deploying RDS cluster and Cumulus ORCA modules in bamboo
+
+For testing, use the `feature/ORCA-test-bamboo` branch in cumulus-orca github repo and use the `ORCA-test-branch` linked repo in bamboo specs. 
+
+While running the `Deploy Dev RDS Stack` stage, replace the following variables with yours. This is because some variables are sensitive and some will vary depending upon the user running the pipeline. Make sure the proper buckets and dynamoDB table are first created manually using the same prefix before running the pipeline. See this cumulus [documentation](https://nasa.github.io/cumulus/docs/deployment/deployment-readme) for additional details.

```

Comment:
Cumulus docs don't specify all the buckets, and the ones they do are split between multiple sections. Recommend pointing to our deployment-with-cumulus.md instead.

Reply:
I'd personally recommend we automate creating any additional resources, and clean them up afterwards. That probably requires a future card.

Reply:
added more info on this in the doc. 

yup already created https://bugs.earthdata.nasa.gov/browse/ORCA-465